src/main/java/com/exadel/discountwebapp/cloud/CloudController.java

```diff
@@ -0,0 +1,23 @@
+package com.exadel.discountwebapp.cloud;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+
+@RestController()
+@RequestMapping("/api/cloud")
+@RequiredArgsConstructor
+public class CloudController {
+
+    private final ImageCloudService imageCloudService;
+
+    @PostMapping("/upload/image")
+    public String uploadImage(@RequestParam("image") MultipartFile image) {
```

Comment:
it's better return the response body with the image id and some params like extension, size

Reply:
done